Makefile

```diff
@@ -579,15 +579,9 @@ contracts/test/prover/proofs/%.json: $(arbitrator_cases)/%.wasm $(prover_bin)
 	@touch $@
 
 .make/cbrotli-wasm: $(DEP_PREDICATE) $(ORDER_ONLY_PREDICATE) .make
-ifdef NODOCKERINDOCKER
-	test -f target/lib-wasm/libbrotlicommon-static.a || ./scripts/build-brotli.sh -w
-	test -f target/lib-wasm/libbrotlienc-static.a || ./scripts/build-brotli.sh -w
-	test -f target/lib-wasm/libbrotlidec-static.a || ./scripts/build-brotli.sh -w
-else
-	test -f target/lib-wasm/libbrotlicommon-static.a || ./scripts/build-brotli.sh -w -d
-	test -f target/lib-wasm/libbrotlienc-static.a || ./scripts/build-brotli.sh -w -d
-	test -f target/lib-wasm/libbrotlidec-static.a || ./scripts/build-brotli.sh -w -d
-endif
+	test -f target/lib-wasm/libbrotlicommon-static.a || ./scripts/build-brotli.sh $(ARGS)
```

Comment:
Almost LGTM but ARGS is way too generic for this, could we call it `CBROTLI_WASM_BUILD_ARGS` ?

Reply:
Also, the default for outside of docker needs to be using docker to build this. The default inside docker can be not using docker-in-docker, that's fine.